### PR TITLE
[bugfix] upgrade transient library commons-beanutils

### PR DESCRIPTION
**Pull request by dizzzz**
- run tests -> OK
- login webdav -> OK

### CODE REVIEW

**Comment by dizzzz** [comment]
sync to 6.x ?

**Review by duncdrum — APPROVED** [review]
Merry x-mas. Nice fix, I have no stake in a backport. @dizzzz would be one less thing where they diverge.

File: extensions/webdav/pom.xml [hunk 1]
```diff
@@ -52,6 +52,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>commons-beanutils</groupId>
```
Comment (adamretter) [inline extensions/webdav/pom.xml:56]: Perhaps add a comment saying why this is hear - as it is transient, it typically should not be here - which means that without extra information, someone could easily remove this in future by mistake.

Comment (dizzzz) [reply extensions/webdav/pom.xml:56]: @adamretter yes makes sense

**Review by reinhapa — APPROVED** [review]

**Pull request merged**